qiskit/circuit/library/phase_oracle.py

```diff
@@ -51,20 +51,23 @@ class PhaseOracle(QuantumCircuit):
     def __init__(
         self,
         expression: Union[str, ClassicalElement],
+        var_order: list = None,
         synthesizer: Optional[Callable[[BooleanExpression], QuantumCircuit]] = None,
```

Comment:
Avoid breaking possible calls by position:
 
```suggestion
        synthesizer: Optional[Callable[[BooleanExpression], QuantumCircuit]] = None,
        var_order: list = None,
```